packages/table/src/quadrants/tableQuadrantStack.tsx

```diff
@@ -265,7 +315,7 @@ export class TableQuadrantStack extends AbstractComponent<ITableQuadrantStackPro
                 <TableQuadrant
                     grid={grid}
                     isRowHeaderShown={isRowHeaderShown}
-                    onWheel={this.throttledHandleWheel}
+                    onWheel={throttleScrolling ? this.throttledHandleWheel : this.handleWheel}
```

Comment:
_definitely_ add a variable for this next to `onMainQuadrantScroll`! 
I see the same ternary statement repeated four times.

Reply:
Ah whoops.